Automattic/jetpack
# Instant Search: Add locale whitelisting

Fixes #14785.

#### Changes proposed in this Pull Request:
* Adds locale whitelisting to ensure only valid locales are passed onto the JavaScript application.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* No.

#### Testing instructions:
1. Follow [these setup instructions](https://github.com/Automattic/jetpack/blob/instant-search-master/modules/search/instant-search/README.md#testing-instructions) to set up Jetpack Instant Search on your site. 

2. Enter a site search. Ensure that the search results appear as expected.

#### Proposed changelog entry for your changes:
* None
